plugins/inventory/stackpath_compute.py

```diff
@@ -0,0 +1,281 @@
+# Copyright (c) 2020 Shay Rybak <shay.rybak@stackpath.com>
+# Copyright (c) 2020 Ansible Project
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = '''
+    name: stackpath_compute
+    plugin_type: inventory
+    short_description: StackPath Edge Computing inventory source
+    version_added: 1.2.0
+    extends_documentation_fragment:
+        - inventory_cache
+        - constructed
+    description:
+        - Get inventory hosts from StackPath Edge Computing.
+        - Uses a YAML configuration file that ends with stackpath_compute.(yml|yaml).
+    options:
+        plugin:
+            description:
+                - A token that ensures this is a source file for the plugin.
+            required: true
+            choices: ['community.general.stackpath_compute']
+        client_id:
+            description:
+                - An OAuth client ID generated from the API Management section of the StackPath customer portal
+                - U(https://control.stackpath.net/api-management).
```

Comment:
```suggestion
                - An OAuth client ID generated from the API Management section of the StackPath customer portal
                  (U(https://control.stackpath.net/api-management)).
```

Reply:
I'm not sure I understand the request here.

all of the links in the repository are in the format I have, and all the descriptions use a list for multiline.

can you elaborate on why this requires changing?

Reply:
Every list item is a paragraph. The first paragraph is a sentence without a period at its end. The second is just an URL with a period. This looks plainly wrong. It should be one paragraph with one sentence. (This is probably also wrong for the other cases this happens in the doc of this plugin, but that's for another PR to fix...)